libraries/botbuilder-ai/src/luis-client/luisClient.ts

```diff
@@ -0,0 +1,200 @@
+/**
+ * @module botbuilder-ai
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+/**
+ * NOTE: This class was auto generated by OpenAPI Generator (https://openapi-generator.tech)
+ * and was manually modified to make it compliant with the current implementation of the library.
+ */
+
+const fetch = (new Function('require', 'if (!this.hasOwnProperty("fetch")) { return require("node-fetch"); } else { return this.fetch; }'))(require);
+
+import * as HttpStatus from 'http-status-codes';
+
+/* tslint:disable:no-unused-locals */
+import { LuisResult } from './model/luisResult';
+
+import { ObjectSerializer, Authentication, VoidAuth } from './model/models';
+import { ApiKeyAuth } from './model/models';
+
+let luisVersion = '/luis/v2.0';
```

Comment:
This should be a `const` for now; it might change in the future but otherwise it should be a `const`

Reply:
Done!